<launch>
<node pkg="drone_project" name="Sequencer" type="Sequencer.py">
</node>
<node pkg="drone_project" name="Main" type="Main.py">
</node>
<node pkg="drone_project" name="optFlow" type="testDenseOpticalFlow.py">
</node>
<node pkg="drone_project" name="curveMotion" type="turn_to_angle.py">
</node>
<node pkg="drone_project" name="turnAng" type="turn_ang.py">
</node>
<node pkg="drone_project" name="computeTarX" type="compute_tar_x.py">
</node>
<node pkg="drone_project" name="computeTarY" type="compute_tar_y.py">
</node>
<node pkg="drone_project" name="computeTarZ" type="compute_tar_z.py">
</node>
<node pkg="drone_project" name="detectVanish" type="image_proc.py">
</node>
<node pkg="drone_project" name="image_processor_drone_vs_stairs_round1" type="stairs_entrance_target.py">
    <remap from="/drone_img" to="/bebop/image_raw/compressed"/>
</node>
<node pkg="drone_project" name="drone_vs_stairs_round1" type="go_to_stairs.py">
</node>

<node pkg="drone_project" name="image_processor_drone_vs_stairs_round2" type="stairs_target_up.py">
    <remap from="/drone_img" to="/bebop/image_raw/compressed"/>
</node>
<node pkg="drone_project" name="drone_vs_stairs_round2" type="get_up_stairs.py">
</node>
</launch>